assets/state-driver-validation/cuda-vector-add-raw.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: nvidia-driver-validation
+  name: nvidia-driver-validation
+  namespace: openshift-sro
+spec:
+#  serviceAccount: nvidia-readonly
+#  serviceAccountName: nvidia-readonly
+  tolerations:
+    - key: nvidia.com/gpu
+      operator: Exists
+      effect: NoSchedule
+  restartPolicy: Never
+  containers:
+    - name: cuda-vector-add
+      image: "quay.io/zvonkok/cuda-vector-add:v0.1"
```

Comment:
Any specific tests that this image has? Or is it just the best one we have so far for the purpose? I like that we use a cuda image rather than just testing nvidia-smi.

Reply:
Only testing nvidia-smi does not involve the cuda library. nvidia-smi has only the NVML binding and cuda is not tested at all when calling it. cuda-vector-add was a compromise between runtime and features. Another thought was to use dcgm diagnostic utilities WDYT?